packages/stylelint-plugin-carbon-tokens/README.md

```diff
@@ -0,0 +1,225 @@
+# stylelint-plugin-carbon-tokens
+
+> A stylelint plugin to support the use of carbon component tokens.
+
+## Getting started
+
+To install `stylelint-plugin-carbon-tokens` in your project, you will need to
+run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S stylelint-plugin-carbon-tokens
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
+instead:
+
+```bash
+yarn add stylelint-plugin-carbon-tokens
+```
+
+## Usage
+
+This project is intended to help users identify cases where tokens, functions
+and mixins defined as part of various projects relating to the Carbon Design
+System are used. [www.carbondesignsystem.com]
+
+It consists of a stylelint plugin and depends on various Carbon Design System
+packages for loading settings.
+
+It includes, but may not be limited to, linting for @carbon/themes,
+@carbon/colors, @carbon/layout, @carbon/type and @carbon/motion.
+
+Not incluced, as they're not used through SCSS, are Carbon Icons, Grid and any
+other DOM related checks..
+
+### Please be helpful
+
+Before we start this project is a work in progress which deliberately outputs
+warnings when it comes accross a syntax that has not yet been catered for. If
```

Comment:
```suggestion
warnings when it comes across syntax that has not yet been catered for. If
```

Reply:
Corrected spelling and grammar, wonder if Grammarly has a VSCode plugin ;-)